.github/workflows/ci.yml

```diff
@@ -415,8 +416,7 @@ jobs:
 
   publish-docker-image:
     needs: cargo-toml-fmt-check
-
-    if: always()
+    if: github.event_name == 'release' && github.event.action == 'published'
```

Comment:
External contributors can't merge PRs because this step requires Fuel org creds. Only Fuel org associates will merge to publish releases.